### PR TITLE
Edited

### PR DESCRIPTION
**Pull request by Paryax**
- [DEMO LINK](https://Paryax.github.io/layout_snake/)
- [TEST REPORT LINK](https://Paryax.github.io/layout_snake/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  }
+
+  .container :nth-child(4) {
+    order: 3;
+  }
+
+}
+
+@media(min-width:900px) {
+
```
Comment (ivan-kadykalo) [inline src/style.css:72]: No empty lines at the start and at the end of the block
```suggestion
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+@media(min-width:600px) {
+
+  .container :nth-child(3) {
+    order: 4;
+  }
+
+  .container :nth-child(4) {
+    order: 3;
+  }
+
```
Comment (ivan-kadykalo) [inline src/style.css:68]:
```suggestion
```

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  background-color: rgb(20%, 0%, 0%);
+}
+
+.container :nth-child(6) {
+  order: 6;
+  background-color: rgb(0%, 0%, 0%);
+}
+
+@media(min-width:600px) {
+
```
Comment (ivan-kadykalo) [inline src/style.css:60]:
```suggestion
```

**Review by ivan-kadykalo — CHANGES_REQUESTED** [review]
Demo links are broken.
Check your links every time.

2 tests in backstop are fallen

File: src/style.css (hunk 1 above)
Comment (Paryax) [reply src/style.css:72]: I have passed all the tests. Links are fixed

**Review by ivan-kadykalo — APPROVED** [review]
GJ🚀

**Pull request closed without merging**